### DISCUSSION TITLE
Why Was the Project Roadmap Removed?

### DISCUSSION THREAD

**Discussion by flavio-evocorp** [usememos/memos#3851] [General]
Hi everyone! I noticed that the project roadmap has been removed and would like to understand why. Could someone explain what happened?

**Comment by boojack** [comment]
Our project management has been moved to Linear instead of the less user-friendly GitHub Project. So there is no public roadmap.

**Reply by flavio-evocorp** [reply]
Got it, thanks a lot for the information.
